### PR TITLE
Responsive improvements to container, submenu, home and displayShow

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

I canceled this change, as it's ugly. And doesn't look nice in the layout. I'm going back to row of buttons, but will add it to a bootstrap row/grid.

Added bs grids to:
- [x] main.mako
- [x] Home
- [x] displayShow
- [x] Recommended Shows (anidb, imdb, trakt)
- [x] fixed submenu.
- [x] Fix introduced rebase conflicts.

### CODE REVIEW

**Comment by p0psicles** [comment]
Still not finished, need to polish some. Or have a real html/css dev finish it. At least the menu feels somewhat usuable on mobile now.

**Comment by p0psicles** [comment]
image

**Comment by fernandog** [comment]
Full HD mobile:
screenshot_2016-10-11-20-08-07

**Comment by p0psicles** [comment]
Aah i missed the scheduled page. Will fix that.

File: static/css/dark.css [hunk 1]
```diff
@@ -284,19 +284,11 @@ ul.tags li {
 .tvshowImg {
     border: 1px solid rgb(17, 17, 17);
     border-radius: 5px;
-    height: 311px;
-    width: auto;
-    float: left;
-}
+ee}
```
Comment (OmgImAlexis) [inline static/css/dark.css]: `ee`?

File: static/css/style.css [hunk 2]
```diff
@@ -3702,3 +3830,81 @@ viewLog.mako
...
+      padding-bottom: 15px;
+      padding-left: 8px;
+  }
+  .mobile .img-align {
+      width: 25px;
+      float: left;
+  }
+}
+
+#SubMenu > a {
```
Comment (OmgImAlexis) [inline static/css/style.css]: We should add a standard to only use lowercase along with hyphens for class names and IDs.

File: views/partials/submenu.mako [hunk 3]
```diff
@@ -1,26 +1,38 @@
...
-              % else:
-                  <a href="${menuItem['path']}" class="btn${('', ' confirm ' + menuItem.get('class', ''))['confirm' in menuItem]}">${('', '<span class="pull-left ' + icon_class + '"></span> ')[bool(icon_class)]}${menuItem['title']}</a>
-                  <% first = False %>
-              % endif
-        % endif
-    % endfor
-    </span>
-</div>
+    <div id="sub-menu-container" class="row">
+	    <div id="SubMenu" class="hidden-print col-md-12">
```
Comment (OmgImAlexis) [inline views/partials/submenu.mako]: This will need to be fixed to be `submenu` or `sub-menu`.

**Comment by p0psicles** [comment]
I'll start replacing camel with hyphens. And fix the typo.

**Comment by duramato** [comment]
```
2016-12-06 23:03:43 DEBUG    Thread-16 :: [b6439a9] File /home/pi/Medusa/medusa/server/web/core/base.py, line 130, in render
2016-12-06 23:03:43 DEBUG    Thread-16 :: [b6439a9] File /home/pi/Medusa/lib/mako/template.py, line 454, in render_unicode
2016-12-06 23:03:43 DEBUG    Thread-16 :: [b6439a9] File /home/pi/Medusa/lib/mako/runtime.py, line 829, in _render
2016-12-06 23:03:43 DEBUG    Thread-16 :: [b6439a9] File /home/pi/Medusa/lib/mako/runtime.py, line 864, in _render_context
2016-12-06 23:03:43 DEBUG    Thread-16 :: [b6439a9] File /home/pi/Medusa/lib/mako/runtime.py, line 890, in _exec_template
2016-12-06 23:03:43 DEBUG    Thread-16 :: [b6439a9] File /home/pi/Medusa/views/layouts/main.mako, line 12, in render_body
2016-12-06 23:03:43 INFO     Thread-16 :: [b6439a9] AttributeError: 'module' object has no attribute 'THEME_NAME'
2016-12-06 23:03:43 ERROR    Thread-16 :: [b6439a9] Exception generated: 'module' object has no attribute 'THEME_NAME'
Traceback (most recent call last):
  File "/home/pi/Medusa/medusa/server/web/core/base.py", line 268, in async_call
    if isinstance(kwargs[arg], binary_type):
  File "/home/pi/Medusa/medusa/server/web/home/handler.py", line 876, in displayShow
    # Only track 5 most recent shows
  File "/home/pi/Medusa/medusa/server/web/core/base.py", line 140, in render
    (name=filename, line=lineno, func=function), logger.DEBUG)
  File "/home/pi/Medusa/lib/mako/template.py", line 454, in render_unicode
    as_unicode=True)
  File "/home/pi/Medusa/lib/mako/runtime.py", line 829, in _render
    **_kwargs_for_callable(callable_, data))
  File "/home/pi/Medusa/lib/mako/runtime.py", line 864, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/home/pi/Medusa/lib/mako/runtime.py", line 890, in _exec_template
    callable_(context, *args, **kwargs)
  File "/home/pi/Medusa/cache/mako/layouts/main.mako.py", line 58, in render_body
    if app.THEME_NAME == "dark":
AttributeError: 'module' object has no attribute 'THEME_NAME'
```

**Comment by p0psicles** [comment]
Yeah because where still adding UI components, this PR is already way outdated with lots of conflicts. These are not to resolve that easy in html.

**Comment by OmgImAlexis** [comment]
Want to move this to the vue repo or keep trying to keep it up to date?

File: views/partials/submenu.mako (hunk 3 above)
Comment (p0psicles) [inline views/partials/submenu.mako]: i'm not going to focus too much on the namings. Yes i know there wrong. But changing these will not matter much for the current UI. And will just cost me allot of time.

**Comment by p0psicles** [comment]
xo: medusa/master has my focus now. As I'd like to have that responsive asap. Then well look if/how we're going to port stuff.

**Comment by p0psicles** [comment]
@duramato that error should be history now. At least I couldn't reproduce.

**Pull request merged**